### DISCUSSION TITLE
Directly use a Raspberry Pi instead of an ESP32 board

### DISCUSSION THREAD

**Discussion by muibusan** [tbnobody/OpenDTU#455] [Q&A] [answered]
Hi!

I have a Raspberry Pi (modell 3 Model B (aarch64)) that is located very closely (1,5 metres) to the microinverter. So, instead of using an ESP32 board, I would love to connect the NRF24L01+ radio board directly to the RPi and let the RPi run all of the needed software.

OpenAhoy offeres a manual page for that at https://github.com/lumapu/ahoy/blob/main/tools/rpi

Is such a setup also feasbile with OpenDTU?

- Is there a wiring scheme for that setup to connect the radio module to the GPIO pins of the RPi?
- What software needs to be installed on the RPi and how?

**Comment by D3R-ST3FAN** [comment, marked as answer]
There is no OpenDTU RPi Port available it  is only available for the ESP32 SoC family.
